finish/pom.xml

```diff
@@ -76,87 +60,52 @@
         <dependency>
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
-            <version>2.0.1</version>
+            <version>2.1.1</version>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+    		<groupId>org.eclipse.microprofile.context-propagation</groupId>
+    		<artifactId>microprofile-context-propagation-api</artifactId>
+    		<version>1.0-RC1</version>
+			<scope>provided</scope>
+		</dependency>
     </dependencies>
     <!-- end::dependencies[] -->
     <!-- tag::build[] -->
     <build>
+		<defaultGoal>clean install liberty:run-server</defaultGoal>
+		<finalName>${project.artifactId}</finalName>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-war-plugin</artifactId>
-                <version>3.1.0</version>
+                <version>${version.maven-war-plugin}</version>
                 <configuration>
                     <packagingExcludes>pom.xml</packagingExcludes>
                 </configuration>
             </plugin>
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>2.0</version>
+                <version>2.3</version>
+
                 <configuration>
-                    <assemblyArtifact>
-                        <groupId>io.openliberty</groupId>
-                        <artifactId>openliberty-runtime</artifactId>
-                        <version>17.0.0.3-RC</version>
-                        <type>zip</type>
-                    </assemblyArtifact>
+					<install>
+						<runtimeUrl>https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/nightly/2019-04-03_0520/openliberty-all-19.0.0.4-cl190420190403-0300.zip</runtimeUrl>
+					</install>
```

Comment:
Should use latest Liberty version once the mpContextPropagation-1.0 feature is GA